File: app/src/main/java/com/annushkaproject/programmerscalculator/model/int_size_enum.java
package com.annushkaproject.programmerscalculator.model;
public enum int_size_enum {
 len8, len4, len2, len1;
 public int wordToInt() {
  switch (this) {
   case len8: return 0; case len4: return 1; case len2: return 2; case len1: return 3;
   default: return -1;
  }
 }
 public static int_size_enum intToWord(int number) {
  switch (number) {
   default: case 0:  return len8; case 1:  return len4;
   case 2:  return len2; case 3:  return len1;   }
 }
}

File: app/src/main/java/com/annushkaproject/programmerscalculator/model/int_size_enum.kt
package com.annushkaproject.programmerscalculator.model

enum class int_size_enum {
    l8, l4, l2, l1;

    fun wordToInt(): Int {
        return when (this) {
            l8 -> 0
            l4 -> 1
            l2 -> 2
            l1 -> 3
            else -> -1
        }
    }

    companion object {
        fun intToWord(number: Int): int_size_enum {
            return when (number) {
                0 -> l8
                1 -> l4
                2 -> l2
                3 -> l1
                else -> l8
            }
        }
    }
}